### PR TITLE
[Task] Create recipe categories asset

### PR DESCRIPTION
**Pull request by Lohenrex**
## Description
Recipes are divided by categories. Whether it is a salad, a pizza or a cake, each on belongs to a different category (e.g. Salads, Italian, Bakery respectively). These asset are managed by an admin.

## Screenshots
- Creating a new recipe category
Creating a new recipe category

- Recipe categories index view
Recipe categories index view

### CODE REVIEW

**Review by itsmeurbi — APPROVED** [review]
Seems to be legit!

File: app/views/recipe_categories/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Recipe categories
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= RecipeCategory.count %> recipe categories!
```
Comment (luisAzcuaga) [inline app/views/recipe_categories/index.html.erb:7]: Avoid doing queries in your views!
Move this to the controller. Also, take a look at [this article](https://medium.com/ruby-daily/comparing-count-length-and-size-in-ruby-on-rails-e061840bc19e) (Actually I was convinced `size` was always the way to go, but It seems that doesn't free us from doing queries to the DB in some scenarios, I personally avoid using count)

Actually you already have  `@recipe_categories` in memory 👀

File: app/views/recipe_categories/index.html.erb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+          </tr>
+        </thead>
+        <tbody>
+          <% @recipe_categories.each do |recipe_category| %>
+            <tr>
+              <td class="has-text-primary">
+                <%= link_to recipe_category.name, edit_recipe_category_path(recipe_category) %>
+              </td>
+              <%= tag.td recipe_category.updated_at %>
+              <%= tag.td recipe_category.created_at %>
```
Comment (luisAzcuaga) [inline app/views/recipe_categories/index.html.erb:31]: ❤️

**Review by luisAzcuaga — CHANGES_REQUESTED** [review]
Other than my suggestion, LGTM! Nice work 😁

File: app/views/recipe_categories/index.html.erb (hunk 1 above)
Comment (Lohenrex) [reply app/views/recipe_categories/index.html.erb]: Thanks a bunch for your feedback, Luis. Already fixed in the latest commit!

**Review by itsmeurbi — APPROVED** [review]
Seems to be legit!
Well done!

File: app/views/recipe_categories/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<div class="column">
+  <h3 class="title has-text-white-ter has-text-right">
+    Recipe categories
+  </h3>
+  <hr class="login-hr">
+  <p class="subtitle has-text-white-ter has-text-right">
+    We already have <%= @recipe_categories.length %> recipe categories!
```
Comment (luisAzcuaga) [inline app/views/recipe_categories/index.html.erb:7]: I saw in the other PR you left this string as interpolated. 
```erb
<%= "We already have #{@variable}..." %>
```
There is no problem whether is this or the other way, just try to be consistent! :)

**Review by luisAzcuaga — APPROVED** [review]
Nice nice! 😁  I'm approving it, just take a look at the note I left

**Pull request merged**